src/MSBuild/LiveLogger/TerminalBuffer.cs

```diff
@@ -100,11 +108,10 @@ public static void Render()
             Console.Write(
                 // Write header
                 ANSIBuilder.Cursor.Home() +
-                ANSIBuilder.Eraser.LineCursorToEnd() + ANSIBuilder.Formatting.Inverse(ANSIBuilder.Alignment.Center("MSBuild - Build in progress")) +
+                ANSIBuilder.Eraser.LineCursorToEnd() + ANSIBuilder.Formatting.Inverse(ANSIBuilder.Alignment.Center($"MSBuild - Build in progress - {FinishedProjects} finished projects")) +
                 // Write footer
-                ANSIBuilder.Eraser.LineCursorToEnd() + ANSIBuilder.Cursor.Position(Console.BufferHeight - 1, 0) +
-                // TODO: Remove and replace with actual footer
-                new string('-', Console.BufferWidth) + $"\nBuild progress: XX%\tTopLineIndex={TopLineIndex}");
+                ANSIBuilder.Cursor.Position(Console.BufferHeight - 1, 0) + ANSIBuilder.Eraser.LineCursorToEnd() +
+                new string('-', Console.BufferWidth) + '\n' + FooterText);
```

Comment:
Don't we want the line endings to be platform specific (`Environment.NewLine`)?
I see it's just adopted from pre-existing code, but unless there is an explicit reason for `\n`, we should use the platform specific line endings.

Reply:
Yes, we should move to this better cross-plat pattern.